internal/btf/btf.go

```diff
@@ -190,6 +190,22 @@ func loadSpecFromELF(file *internal.SafeELFFile) (*Spec, error) {
 	return spec, nil
 }
 
+// GetFuncLineInfos returns a map of LineInfo's for the given function name. The map is indexes by the
+// instruction offset of the lineInfo.
+func (spec *Spec) GetFuncLineInfos(funcName string) map[uint32]LineInfo {
```

Comment:
Can you use `Spec.Program` and `Program.LineInfos` instead? The implementation of this function can be private to the elf reader.

Reply:
Is that the direction we want to be headed in?, especially regarding #522

Reply:
It's better than adding new API which we need to take care of later, and it doesn't necessarily make #522 harder I think.